tests/e2e/integration/amount-send.js

```diff
@@ -1,7 +1,9 @@
 describe('Test cases AmountSend component', () => {
   it('Calculates currency on enter amount, validates entered amount, shows correct balance', () => {
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
     cy.login()
       .openWithdraw()
+      .wait(1000)
       .get('[data-cy=balance-currency]')
       .invoke('text')
       .then((text) => expect(text).not.to.eq('$0.00'))
```

Comment:
```suggestion
    cy.login()
      .openWithdraw()
      .get('[data-cy=balance-currency]')
      .invoke('text')
      .should('not.equal', '$0.00')
```

Reply:
It won't work since text are not trimmed.
https://github.com/cypress-io/cypress/issues/3887